lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -45,7 +45,8 @@
         required: true
     privatekey_passphrase:
         description:
-            - The passphrase for the privatekey.
+            - The passphrase for the privatekey(privatekey_path).
```

Comment:
I think you added a little too much here:
```suggestion
            - The passphrase for the privatekey.
```